app/service/data_svc.py

```diff
@@ -49,8 +49,14 @@ def __init__(self, dao):
             await self.dao.create('core_parser', parser)
         return 'Saved ability: %s' % id
 
-    async def create_adversary(self, name, description, phases):
-        identifier = await self.dao.create('core_adversary', dict(name=name.lower(), description=description))
+    async def create_adversary(self, name, description, phases, locked, init=False):
```

Comment:
i get where you're going with the init flag but i want to try to accomplish the same thing with just the 'locked' flag. when you get the adversary (line 53) we could check if it exists and is/isn't locked before proceeding- i think that may do the trick.

Reply:
While that would absolutely work, the idea was that reading from files overwrites locking entirely. If we go down the route you suggest, that would no longer be the case, which will work perfectly when we fully flesh this out, but in the mean time, I'm a little more hesitant to lock things permanently short of a database wipe.

Reply:
you are maintaining the reading-from-files overwriting locking by ensuring load_adversaries() passes the locked/unlocked flag to create_adversary(). the load_* functions are only used during the booting up of the server (init) so we can piggyback on that to replace the init flag itself. 

the key thing i want to get here is to add a single parameter to the create_adversary() to determine if it will be locked or unlocked.